### DISCUSSION TITLE
Requirements for highest possible config

### DISCUSSION THREAD

**Discussion by jadasse** [86Box/86Box#1537] [Q&A] [answered]
I know the main page says requirements vary based on the config, so, is there a set of recommended host requirements for the highest, most advanced configuration for 86box? For instance, an (Intel Celeron (Mendocino) 533, 2gb ram, 3dfx voodoo3 3000?) Is the 11980hk a good example?

**Comment by bachimusprime** [comment, marked as answer]
No computer currently available will be able to run that configuration at 100% reliably. It's just too demanding.

**Reply by jadasse** [reply]
Oh ok

**Reply by bachimusprime** [reply]
Oh, and furthermore, the setup you listed isn't even the most intensive setup. You'd have to change the Pentium II 450 to a Celeron Mendocino 533.

**Reply by jadasse** [reply]
I thought so

**Reply by jadasse** [reply]
By the way, would an i9 11980hk be suitable for a celeron 533?

**Reply by bachimusprime** [reply]
Nope, that's a laptop CPU if I'm not mistaken, and those tend to be WAY less powerful than desktop CPUs. You're better off waiting a couple years before getting your hopes up :P

**Reply by jadasse** [reply]
> Nope, that's a laptop CPU if I'm not mistaken, and those tend to be WAY less powerful than desktop CPUs. You're better off waiting a couple years before getting your hopes up :P

I was just curious as I honestly don't own one
